cursorless-talon/src/actions/actions_simple.py

```diff
@@ -20,6 +20,7 @@
     "extract": "extractVariable",
     "float": "insertEmptyLineAfter",
     "fold": "foldRegion",
+    "follow": "follow",
```

Comment:
Should we rename the action to `followLink`?  Might be a bit clearer in the cheatsheet, without making it so specific that we shoot ourselves in the foot down the road

Reply:
Kind of?  It is following a link to somewhere else.  I would argue that our usage of "link" here is short for "hyperlink", which is defined as

> a link from a hypertext document to another location, activated by clicking on a highlighted word or image

Given that we're just mimicking the default VSCode ctrl-click behaviour, I think this definition applies?  Idk, I also think if someone just sees "follow" they'll be confused

Reply:
You and I know that but I would be surprised if people would consider that a link in general
Do you want follow link I have no objections

Reply:
I'd lean towards switching to `followLink` if you don't have a strong preference, just because if I saw "follow", I'm not sure I'd know what it was supposed to do.  I certainly wouldn't be likely to assume that it would reveal definition, which is the reason that we would use it instead of `followLink`, right?

Reply:
followLink it is